i18n/fr/docusaurus-plugin-content-docs/version-24.04/installation/installation-of-a-central-server/using-packages.md

```diff
@@ -315,7 +315,9 @@ Installez les dépendances suivantes :
 apt update && apt install lsb-release ca-certificates apt-transport-https software-properties-common wget gnupg2 curl
 ```
 
-#### Installer le dépôt Sury APT pour PHP 8.1
+#### Installer le dépôt Sury APT pour PHP 8.1 
+
+>Cette étape n'est nécessaire que pour les utilisateurs sur Debian 11, si vous êtes sur Debian 12, passez à l'étape suivante.
```

Comment:
Centreon 24.04 and packages require PHP 8.1 and installation of 8.2 is not automatic, so Centreon packages are not required for this version.
It's not OK actually.

Reply:
@emgcentreon the target versions for this ticket did not include version 24.04.